pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+## Links
+
+- [Jira task]()
+- [Upstream repository]()
+- [Changelog/diff]()
+- [Related PR(s)]()
+
+--------------------
+
+## Description
+
+- Target channel: 
+- Package version: 
+### Dependency chain
+N/A
+
+### Information checked from upstream: 
+
+- [ ] All dependency pinnings match with upstream.
+- [ ] All URLs are correct.
+- [ ] abs.yaml is not present at time of review.
```

Comment:
Sometimes it makes sense to have it present at time of review, for example snowflake/non-defaults deliveries.

Reply:
Doesn't it upload to the channel as soon as you add the abs.yaml though?